environments/v2f-prod/terraform/backend.tf

```diff
@@ -0,0 +1,10 @@
+###
+## Store all of our state in GCS so we have a single
+## source of truth.
+###
+terraform {
+  backend "gcs" {
+    bucket = "broad-dsp-monster-prod-terraform-state"
+    path = "v2f.json"
```

Comment:
nitpick: would this be clearer if it was `v2f-state.json` or something with the word state?